money.go

```diff
@@ -85,6 +88,33 @@ func New(amount int64, code string) *Money {
 	}
 }
 
+// NewFromFloat creates and returns new instance of Money from a float64.
+func NewFromFloat(amount float64, currency string) *Money {
```

Comment:
Should we mention in the comments how rounding works?
Please also add new function to README